service/src/main/java/bio/terra/pipelines/stairway/imputation/AddWdsRowStep.java

```diff
@@ -80,8 +80,7 @@ public StepResult doStep(FlightContext flightContext)
           flightContext.getFlightId(), // this is the primary key for WDS
           "flight_id");
     } catch (WdsServiceException e) {
-      // not sure what exception makes sense to throw here
-      return new StepResult(StepStatus.STEP_RESULT_FAILURE_FATAL, e);
+      throw new RetryException("Error creating record in WDS. Will retry.", e);
```

Comment:
does the retryexception log the stack trace or anything before retrying?

Reply:
Stairway should log it, yes: https://github.com/DataBiosphere/stairway/blob/develop/stairway/src/main/java/bio/terra/stairway/impl/FlightRunner.java#L296